### ISSUE TITLE
Following, Known followers, Aware of follow, Confirmed friends

### ISSUE THREAD

**Issue by johlrogge** [agical/friendsunhosted#17]
When you add a _friend_ the friend won't know that this happened unless he already has you as a friend so in that case you are just "**following** the person.

When the _followed_ person adds you the follow is a **mutual follow**

You may not want to follow everyone that follows you (like on twitter) but it would be nice to acknowledge that you are aware of the follow by adding the friend to **known followers**

I may claim that a person I follow is my _friend_, _spouse_ etc. When I do this will be listed as unconfirmed until there is a signed confirmation in the friend's store.

**Comment by jancborchardt** [comment]
Hmm, I’m not sure if an addition to the follow system doesn’t overcomplicate the way friends are handled. I’d say we should start out simple, with a follow-follower-mutualfollow system like Twitter. Mutual follows are then higher prioritized in the friends list.

**Comment by johlrogge** [comment]
The users don't have to be bothered with all the details. For instance, known followers can be added automatically.

My biggest issue is that we call the site _friends_ unhosted which is sort of the original intention. However our system is actually just followers, and in addition to that they are unknown to the followee, more like "stalkers" :)

Followers would become **known** as a sideeffect of #16 (if the followed person is reachable withing the TTL of the message).  It would give the feedback that the follow has been seen (or not).

To be able to specify what your relationship is (ie friend, spouse, collegue) is more in line with linked in or facebook. It would be nice if it could scale from twitter to FB/LInked In seamlessly. To be able to confirm this relationship is needed since unless confirmed it cannot be trusted information.

I don't think that the problem is that it would be too much information. The issue is more to ease the user in from a pretty anonymous unspecified graph to optionally add details to the graph.

What is really cool with F#Un is that the graph would be detailed to people with in your graph. There is no central authority that collects the data about all the citizens in the graph. It is there for the benefit of the users in th epower of the users.

**Comment by johlrogge** [comment]
The relationship data could easily be encrypted for unconfirmed people so you can only see what type of relationship two people have if you are a trusted member of the graph (with asymetrical + symetrical encryption). Agai, a user must not really know that it is solved with asymetrical + symetrical encryption. Just make a policy choice who will see the information (and respect their friends chosen policies)

**Comment by jancborchardt** [comment]
Yeah, for this we should definitely also use the remoteStorage contacts category/module. Then it can also be used by [Meute](https://github.com/pirates-of-the-dark-table/meute) by @michielbdejong @nilclass, @xmartin and @azul, as well as http://sharedstuff.org by @shybyte.
